homework-02-relaxr-landing-page/css/style.css

```diff
@@ -0,0 +1,196 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: Helvetica, Arial, sans-serif;
+     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+    background-size: cover;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 960px;
+  margin: auto;
+  height: 500px;
+}
+
+header {
+
+}
+
+footer {
+  background-color: #121212;
+  padding: 30px;
+  text-align: center;
+
+}
+
+/******************************************
+/* ADDITIONAL STYLES
+/*******************************************/
+
+h1 { 
+  font-size: 24px;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h2 {
+  font-size: 18px;
+}
+.txt-white  {
+  color: #fff;
+}
+
+.txt-yellow{
+  color: yellow;
+}
+
+.top-bar {
+  padding-top: 20px;
+  padding-left: 20px;
+  font-weight: 800;
+}
+
+.tag_line {
+  text-align: center;
+  padding: 30px;
+  font-weight: 500;
+}
+
+.shop1{
+  padding: 20px 20px 200px 20px;
+  text-align: center;
+}
+
+
+#button1 {
+  width: 300px;
+  background-color: #f9e42e;
+  color: #033048;
+  font-weight: 800;
+  font-size: 14px;
+  border-radius: 5px;
+  border: none;
+  padding: 20px;
+  margin: 0 auto;
+}
+
+.benefits{
+  background-color: #f0efef; 
+  color: #606161;
+  padding: 30px 20px;
+  font-size: 13px;
+  font-weight: 300;
+}
+
+.benefits h2 {
+  font-weight: 500;
+  color: #033048;
+}
+
+
+.benefits li:before {
```

Comment:
@Uliana202 Very resourceful solution to get the bullet points you wanted! Awesome job here.